mapshader/flask_app.py

```diff
@@ -113,7 +113,19 @@ def build_previewer(service: MapService):
     p.axis.visible = True
 
     if service.service_type == 'tile':
+        tile_source = WMTSTileSource(url=service.client_url,
+                                     min_zoom=0,
+                                     max_zoom=15)
 
+        p.add_tile(tile_source, render_parents=False)
+
+    elif service.service_type == 'geojson':
+        geo_source = GeoJSONDataSource(geojson=render_geojson(service.source))
```

Comment:
Issue #53 is about adding point, line, polygon and GeometryCollection support but the data from the world_countries_source is a FeatureCollection, how does that relate?

Reply:
@IuryPiva , the world_countries_source data source loads from the `naturalearth_lowres` dataset, which if you run the following code, you will see that it has the Polygon geometric object.

```
>>> import geopandas as gpd
>>> data_gdf = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
>>> data.geom_type.unique()
array(['MultiPolygon', 'Polygon'], dtype=object)
>>> print(data["geometry"][0])
MULTIPOLYGON (((180 -16.06713266364245, 180 -16.5552165666392, 179.3641426619641 -16.80135407694688, 178.7250593629971 -17.01204167436804, 178.5968385951171 -16.63915, 179.0966093629971 -16.4339842775474, 179.4135093629971 -16.3790542775474, 180 -16.06713266364245)), ((178.12557 -17.50481, 178.3736 -17.33992, 178.71806 -17.62846, 178.55271 -18.15059, 177.93266 -18.28799, 177.38146 -18.16432, 177.28504 -17.72465, 177.67087 -17.38114, 178.12557 -17.50481)), ((-179.7933201090486 -16.02088225674122, -179.9173693847653 -16.5017831356494, -180 -16.5552165666392, -180 -16.06713266364245, -179.7933201090486 -16.02088225674122)))
```

You can understand better by taking a look at https://geopandas.org/docs/user_guide/data_structures.html.

![Screenshot from 2021-04-21 16-24-06](https://user-images.githubusercontent.com/16704806/115609630-28fbab00-a2be-11eb-9834-7df77924b690.png)

Reply:
If I understood correctly, Bokeh supports this types in the same DataSource as the FeatureCollection. But I have not been able to get a GeometryCollection GeoJSON from our sample sources =( Any ideas?

Reply:
But based on this answer on SO https://stackoverflow.com/a/34045862 I think we achieved the objective of this PR